x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -96,13 +100,35 @@ export type FilterItems = Array<{
 
 // TODO: add more filter names here
 export type FilterName = keyof typeof FILTER_NAMES;
-export const useActionsLogFilter = () => {
+export const useActionsLogFilter = (
+  filterName: FilterName
+): {
+  items: FilterItems;
+  setItems: React.Dispatch<React.SetStateAction<FilterItems>>;
+  hasActiveFilters: boolean;
+  numActiveFilters: number;
+  numFilters: number;
+} => {
+  const isStatusesFilter = filterName === 'statuses';
   const [items, setItems] = useState<FilterItems>(
-    RESPONSE_ACTION_COMMANDS.slice().map((filter) => ({
-      key: filter,
-      label: filter === 'unisolate' ? 'release' : filter,
-      checked: undefined,
-    }))
+    isStatusesFilter
+      ? RESPONSE_ACTION_STATUS.map((filter) => ({
+          key: filter,
+          label: (
+            <StatusBadge
+              color={
+                filter === 'successful' ? 'success' : filter === 'failed' ? 'danger' : 'warning'
+              }
+              status={filter.slice(0, 1).toUpperCase().concat(filter.slice(1))}
```

Comment:
Did you forget about this one here?
if `status` is being displayed on the UI, you should not be capitalizing it and showing it as is - it needs to be `i18n`'d

Reply:
524c760b6a3cc8ab4728a85c77cbfca85e2bad45